core/src/main/java/org/stellar/anchor/sep6/Sep6Service.java

```diff
@@ -107,10 +107,11 @@ public StartDepositResponse deposit(Sep10Jwt token, StartDepositRequest request)
     }
 
     StellarAssetInfo asset = requestValidator.getDepositAsset(request.getAssetCode());
-    if (request.getType() != null) {
-      requestValidator.validateTypes(
-          request.getType(), asset.getCode(), asset.getSep6().getDeposit().getMethods());
-    }
+    String fundingMethod =
+        request.getFundingMethod() != null ? request.getFundingMethod() : request.getType();
```

Comment:
Should we reject requests with `funding_method` and `type` specified?

Reply:
I wouldn't worry about that. If a wallet is using `funding_method ` I believe they have read the SEP and it's clearly stated that `type` is deprecated in favor of `funding_method`.